### PR TITLE
Swap unzip to nf-test (continuation of 5357)

### PR DESCRIPTION
**Pull request by SPPearce**
Continuing #5357 as I can't update the branch there.

### CODE REVIEW

**Comment by SPPearce** [comment]
Hmmm, malt seems to be failing to get the input file from aws s3....
One for the task force I guess @jfy133

**Comment by jfy133** [comment]
@nf-core/core do we need special tokens or keys on the modules repo to be available to access S3 stuff?

**Comment by maxulysse** [comment]
> @nf-core/core do we need special tokens or keys on the modules repo to be available to access S3 stuff?

What do you need?

**Comment by SPPearce** [comment]
> > @nf-core/core do we need special tokens or keys on the modules repo to be available to access S3 stuff?
> 
> What do you need?

The tests are failing for the malt modules, as they have some test data on s3

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/maltextract/tests/main.nf.test [hunk 1]
```diff
@@ -25,8 +25,9 @@ nextflow_process {
             process {
                 """
                 input[0] = [ [], // meta map
-                file('https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/malt/test.rma6') ]
-                input[1] = file('https://github.com/nf-core/test-datasets/raw/modules/data/genomics/sarscov2/genome/db/maltextract/taxon_list.txt')
+                    file(params.modules_testdata_base_path + 'delete_me/malt/test.rma6')
+                ]
+                input[1] = file(params.modules_testdata_base_path + 'genome/db/maltextract/taxon_list.txt')
```
Comment (maxulysse) [inline modules/nf-core/maltextract/tests/main.nf.test:30]:
```suggestion
                input[1] = file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/db/maltextract/taxon_list.txt')
```

File: modules/nf-core/malt/build/tests/main.nf.test
Comment (maxulysse) [inline modules/nf-core/malt/build/tests/main.nf.test:1]: so the input for unzip on line 19 is a `739.0 MiB` zip...
I'm guessing that's why the tests are failing...

Comment (maxulysse) [reply modules/nf-core/malt/build/tests/main.nf.test:1]: cf https://github.com/nf-core/modules/pull/4801

**Pull request merged**